modules/transcription-service-google-speech-impl/src/main/java/org/opencastproject/transcription/googlespeech/GoogleSpeechTranscriptionService.java

```diff
@@ -1014,7 +1048,7 @@ public void run() {
             // Apply workflow to attach transcripts
```

Comment:
"Jobs that get here have state TranscriptionCompleted [... or had an IOException]"